docker-storage-setup.1

```diff
@@ -98,67 +100,34 @@ WIPE_SIGNATURES:
       overwritten as suitable. This also means that if there is any
       data on disk, it will be lost.
 
-DOCKER_ROOT_VOLUME:
-      By default no new volume and filesystem will be setup
-      for docker root dir. Docker creates /var/lib/docker/ on
-      top of underlying filesystem for storing images and containers.
-
-      To carve out a separate logical volume for storing docker
-      images/containers/volumes data, set DOCKER_ROOT_VOLUME=yes
-
-      docker-storage-setup creates a logical volume with an XFS
-      filesystem mounted on docker root directory
-      (default: /var/lib/docker).
-
-      Note: DOCKER_ROOT_VOLUME is deprecated, and will be removed
-      soon. Use CONTAINER_ROOT_LV_MOUNT_PATH instead.
-
-      Note: A user cannot specify DOCKER_ROOT_VOLUME and
-      CONTAINER_ROOT_LV_MOUNT_PATH together. They are mutually exclusive
-      options.
-
-DOCKER_ROOT_VOLUME_SIZE:
-      Specify the desired size for docker root volume. It defaults
-      to 40% of all free space.
-
-      DOCKER_ROOT_VOLUME_SIZE can take values acceptable to
-      "lvcreate -L" as well as some values acceptable to to 
-      "lvcreate -l". If user intends to pass values acceptable 
-      to "lvcreate -l", then only those values which contains "%" 
-      in syntax are acceptable.  If value does not contain "%" it 
-      is assumed value is suitable for "lvcreate -L".
-
-      Note: If both STORAGE_DRIVER=devicemapper and
-      DOCKER_ROOT_VOLUME=yes is set, docker-storage-setup would set
-      up the thin pool for devicemapper first, followed by docker 
-      root volume. e.g if free space in the volume group is 10G, 
-      devicemapper thin pool size would be 4G (40% of 10G) and docker
-      root volume would be 2.4G (40% of 6G).
-
 CONTAINER_ROOT_LV_NAME:
```

Comment:
We are getting rid of DOCKER_ROOT_VOLUME_SIZE. That means we need to output an Info message saying this is deprecated. I don't see that in code yet.